src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -112,25 +112,19 @@
     <None Remove="Views\Menu\TrustedPathView.xaml" />
     <None Remove="Views\GuidedTour\HtmlPages\Resources\ConnectTheNode.gif" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
-    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" /> 
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' "> 
-    <Reference Include="System.Configuration" />
-    <Reference Include="PresentationCore" />
-    <Reference Include="PresentationFramework" />
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Drawing" />
-    <Reference Include="System.Runtime.Serialization" />
-    <Reference Include="System.Web" />
-    <Reference Include="System.Windows.Forms" />
-    <Reference Include="System.Xaml" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Xml" />
-    <Reference Include="WindowsBase" />
-  </ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
+        <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
+        <PackageReference Include="HelixToolkit" Version="2.24.0" />
+        <PackageReference Include="HelixToolkit.Core.Wpf" Version="2.24.0" />
+        <PackageReference Include="HelixToolkit.SharpDX.Core.Wpf" Version="2.24.0" />
```

Comment:
HelixToolkit.Core.Wpf and HelixToolkit.SharpDX.Core.Wpf are only targeting netcoreapp3.1
We can either make sure (as much as we can by building locally, testing etc) that these are compatible with net6 or we try to get a proper net6 nuget (contribute to the open source or fork it) 
@mjkkirschner @sm6srw @aparajit-pratap thoughts ?

Reply:
I think the maintainers of helix are quite responsive - but I know the project is complex, considering the background geometry preview is a core component, it would be ideal if we could ensure it works - we don't have bandwidth or time to move to something else now - it seems the testing time could be spent working on the migration to net6 and if they don't accept it we can always use the fork locally until we can get it merged or replace with something else (usd, webgpu, etc etc) that is cross platform.

I think a spike ascertaining how difficult it is makes sense before making this decision.

Reply:
perhaps in the meantime we can merge this and @avidit can manually evaluate the image comparison tests to make sure they are the same in net6 buids?

Reply:
@jasonstratton @pinzart90 @marbelarrietaGlobant ... seems we are running out of time to try these alternative approaches that are discussed here.

@marbelarrietaGlobant what is the state of this PR?

Reply:
@mjkkirschner issues with the PR were solved. Just waiting to approve and merge or another revision.